packages/dd-trace/src/plugins/database.js

```diff
@@ -38,13 +38,17 @@ class DatabasePlugin extends StoragePlugin {
   }
 
   injectDbmQuery (query, serviceName, isPreparedStatement = false) {
-    if (this.config.dbmPropagationMode === 'disabled') {
+    const dbmPropagationMode = this.getConfig('dbmPropagationMode')
```

Comment:
This is an example of how plugins should be loading overridable configuration options.

Reply:
This is adding overhead in possibly hot paths.

Reply:
I've removed getConfig